src/main/twirl/gitbucket/core/helper/datetimeago.scala.html

```diff
@@ -1,5 +1,5 @@
 @(latestUpdatedDate: java.util.Date,
-  recentOnly: Boolean = true)
+  recentOnly: Boolean = true)(implicit context: gitbucket.core.controller.Context)
```

Comment:
It might be not necessary to provide this as Twirl template. How about moving this to helper?

Reply:
It means "move to `gitbucket.core.view.helpers`?"

Reply:
Yes, but I think we might have to consider about behavior of date formatting again because it has little complex behavior currently. For example, `datetimeAgoRecentOnly` returns absolute date for old dates in current implementation.

In addition, I think `Date.toString()` is not good solution to generate display strings.